integr8ly/integreatly-operator
# [INTLY-2333] Nexus install

### Description 
Add nexus operator 
Manifest PR: https://github.com/integr8ly/manifests/pull/15

### Verification steps
- Create the operator source which points to my app registry:
```yml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators
  endpoint: 'https://quay.io/cnr'
  publisher: Integreatly Publisher
  registryNamespace: dimitra
  type: appregistry
```
- Create a subscription to the Integreatly operator from the Operator Hub, then create an `installation` CR 
```yml
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: workshop
  namespacePrefix: integreatly-
```
- Verify that the `integreatly-nexus` namespace is created
- Verify that the `nexus-operator` pod and `nexus` application pod are running in that namespace 
- Verify that the nexus route is accessible 
- Verify that when deleting any of these components (the nexus namespace, subscription, nexus custom resource, application pod, etc) they are eventually recreated
